Packs/CheckPointHEC/ReleaseNotes/1_1_0.json

```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": null
```

Comment:
Please explain in breakingChangesNotes what had been changed that resulted in BC.
for example: https://github.com/demisto/content/blob/master/Packs/AWS-AccessAnalyzer/ReleaseNotes/1_1_0.json